Resources/views/MediaAdmin/multi_upload.html.twig

```diff
@@ -0,0 +1,137 @@
+{% extends 'SonataAdminBundle:CRUD:base_edit.html.twig' %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    <link rel="stylesheet" href="{{ asset('bundles/sonatamedia/multi_upload/css/jquery.fileupload.css') }}">
+    <link rel="stylesheet" href="{{ asset('bundles/sonatamedia/multi_upload/css/jquery.fileupload-ui.css') }}">
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    <script src="{{ asset('bundles/sonatamedia/multi_upload/js/load-image.all.min.js') }}"></script>

```

Comment:
but we only need them when we use the multi uploader and don't want to load them all the time IMO